server/server.go

```diff
@@ -0,0 +1,46 @@
+package server
+
+import (
+	"fmt"
+	"github.com/jinzhu/gorm"
+	"github.com/kirikami/go_exercise_api/config"
+	"github.com/kirikami/go_exercise_api/routes"
+	"github.com/labstack/echo"
+	"github.com/labstack/echo/engine/standard"
+	"github.com/labstack/echo/middleware"
+)
+
+func StartServer(db *gorm.DB, appConfig *config.Configuration) {
+	port := fmt.Sprintf(":%d", appConfig.ListenAddress)
+	server := echo.New()
+
+	server.Use(middleware.Recover())
+	server.Use(middleware.Logger())
+	server.Use(middleware.CORSWithConfig(middleware.CORSConfig{
+		AllowHeaders: []string{echo.HeaderAccessControlAllowOrigin},
+	}))
+
+	api := routes.ApiV1Handler{db, appConfig}
+
+	v1 := server.Group("/v1")
+
+	aut := v1.Group("/auth")
+	aut.GET("", api.AutenteficationHandler)
+	aut.GET("/callback", api.ProviderCallback)
+
+	task := v1.Group("/task")
+	task.Use(middleware.JWT([]byte(appConfig.SigningKey)))
+
+	task.POST("/save", api.SaveTaskHandler)
```

Comment:
This is not REST style, 
Add task, use `POST /task`
Update task, use `PUT /task/:id`
Delete task, use `DELETE /task/:id`
List of tasks `GET /tasks`